src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/InteropServices/PreferIsExactSpellingIsTrueForKnownApisTest.cs

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Testing;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.InteropServices.PreferIsExactSpellingIsTrueForKnownApisAnalyzer,
+    Microsoft.NetCore.CSharp.Analyzers.InteropServices.CSharpPreferIsExactSpellingIsTrueForKnownApisFixer>;
+
+namespace Microsoft.NetCore.Analyzers.InteropServices.UnitTests
+{
+    public sealed class PreferIsExactSpellingIsTrueForKnownApisTest
+    {
+        private DiagnosticResult CA1839_DefaultRule(int markupId, string knownApiName)
+           => VerifyCS.Diagnostic(PreferIsExactSpellingIsTrueForKnownApisAnalyzer.DefaultRule)
+               .WithLocation(markupId)
+               .WithArguments(knownApiName);
+
+        private DiagnosticResult CA1839_WideRule(int markupId, string knownApiName)
+           => VerifyCS.Diagnostic(PreferIsExactSpellingIsTrueForKnownApisAnalyzer.WideRule)
+               .WithLocation(markupId)
+               .WithArguments(knownApiName);
+
+        [Fact]
+        public async Task CA1839CSharpTest()
```

Comment:
Could you please split this test into multiple? Like no diagnostic test, ExactSpelling missing, A suffix...